services-directions/src/main/java/com/mapbox/api/directions/v5/WalkingOptions.java

```diff
@@ -0,0 +1,152 @@
+package com.mapbox.api.directions.v5;
+
+import android.support.annotation.FloatRange;
+import android.support.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+
+/**
+ * Class for specifying options for use with the walking profile.
+ * @since 4.8.0
+ */
+@AutoValue
+public abstract class WalkingOptions {
+
+  /**
+   * Walking speed in meters per second. Must be between 0.14 and 6.94 meters per second.
+   * Defaults to 1.42 meters per second
+   *
+   * @return walkingSpeed in meters per second
+   * @since 4.8.0
+   */
+  @Nullable
+  public abstract Double walkingSpeed();
```

Comment:
Shouldn't options include the `@SerializedName`? That’s what the backend expects / generates

Reply:
👀 https://github.com/mapbox/mapbox-java/pull/991#discussion_r282667172